e2e/nomostest/clean.go

```diff
@@ -190,7 +189,7 @@ func uninstallUnmanagedPackagesOfType(nt *NT, gvk schema.GroupVersionKind) error
 	// Find the unmanaged packages
 	var rsObjs []*unstructured.Unstructured
 	for _, item := range rsObjList.Items {
-		if core.GetAnnotation(&item, metadata.ResourceManagementKey) != metadata.ResourceManagementEnabled {
+		if !metadata.IsManagementEnabled(&item) {
```

Comment:
Is the reason this uses `!metadata.IsManagementEnabled` rather than `metadata.IsManagementDisabled` to catch the default case where the annotation is unset?

Reply:
Might read better if the cases were flipped, but it's skipping managed RSyncs. So all the RSyncs on the cluster without management enabled must have been manually applied.

The annotation should never be set to any other value on the cluster. If the user sets it to disabled in the source, the annotation is removed from the previously applied object.

Reply:
Swapped